ext/drcovlib/modules.c

```diff
@@ -234,7 +238,7 @@ static inline void
 lookup_helper_set_fields(module_entry_t *entry, OUT uint *mod_index, OUT app_pc *mod_base)
 {
     if (mod_index != NULL)
-        *mod_index = entry->id;
+        *mod_index = entry->id; /* We expose the segment. */
```

Comment:
It has to be exposed: some users will not work properly otherwise.  I think you were not aware of what is exposed: see drmodtrack_info_t's docs.